app/views/invoices/edit.erb

```diff
@@ -1,7 +1,13 @@
 <form action="/invoices/<%= @invoice.id %>" method="post">
   <input type='hidden' name='_method' value='put'>
-  <p>Edit</p>
-    <div>Status</div>
+  <p>Invoice: <%= @invoice.id%> - <%= @invoice.status %></p>
+  <div>Merchant</div>
+    <!-- <select class="merchant-name text-box" name="merchant[name]" disabled> -->
```

Comment:
@hbellows does this need to have the comment for the disabled drop down to work?